scripts/build_go.sh

```diff
@@ -93,6 +98,20 @@ if [[ "$sign_darwin" == 1 ]]; then
 	requiredenvs AC_CERTIFICATE_FILE AC_CERTIFICATE_PASSWORD_FILE
 fi
 
+if [[ "$sign_windows" == 1 ]]; then
+	dependencies osslsigncode
+	requiredenvs AUTHENTICODE_CERTIFICATE_FILE AUTHENTICODE_CERTIFICATE_PASSWORD_FILE
+fi
+
+if [[ "$os" == "windows" ]]; then
+	goversioninfo 	-platform-specific=true \
```

Comment:
Currently the following files are outputted in `~`

```
-rw-r--r--  1 coder coder 193962 Nov 23 05:07 resource_windows_386.syso
-rw-r--r--  1 coder coder 193962 Nov 23 05:07 resource_windows_amd64.syso
-rw-r--r--  1 coder coder 193962 Nov 23 05:07 resource_windows_arm.syso
-rw-r--r--  1 coder coder 193962 Nov 23 05:07 resource_windows_arm64.syso
```

Need to embed the appropriate arch into the appropriate windows build.